### PR TITLE
Jupyter nested columns tutorial

### PR DESCRIPTION
**Pull request by writer-jill**
Jupyter notebook tutorial - working with nested columns.
This PR has:
- [x] been self-reviewed.

### CODE REVIEW

**Comment by writer-jill** [comment]
PR is out of date - closing and will open a new one.

File: examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+    "  ~ software distributed under the License is distributed on an\n",
+    "  ~ \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY\n",
+    "  ~ KIND, either express or implied.  See the License for the\n",
+    "  ~ specific language governing permissions and limitations\n",
+    "  ~ under the License.\n",
+    "  -->\n",
+    "\n",
+    "This tutorial demonstrates how to work with [nested columns](https://druid.apache.org/docs/latest/querying/nested-columns.html) in Apache Druid.\n",
+    "\n",
+    "Druid stores nested data structures in `COMPLEX<json>` columns. In this tutorial you perform the following tasks:\n",
```
Comment (techdocsmith) [inline examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb:30]: I like this intro. I wonder if there is a way to make a suggestion for this type of intro in the template

File: examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb [hunk 2]
```diff
@@ -0,0 +1,434 @@
...
+    "- Use helper operators to examine nested data and plan your queries.\n",
+    "\n",
+    "Druid supports directly ingesting nested data with the following formats: JSON, Parquet, Avro, ORC, Protobuf."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Table of contents\n",
```
Comment (techdocsmith) [inline examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb:44]: I like the TOC. Can we add this to the template?

File: examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+   "metadata": {},
+   "source": [
+    "## Initialization"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run the following cell to initialize the environment for the tutorial. The quickstart deployment configures Druid to listen on port `8888` by default, so you'll make API calls against `http://localhost:8888`."
```
Comment (techdocsmith) [inline examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb:84]: This isn't the case and the template doesn't explain it.

```suggestion
    "Run the following cell to initialize the environment for the tutorial. Docker Compose automatically sets the DRUID_HOST for you based upon your profile. If DRUID_HOST isn't found, the script connects to druid on http://localhost:8888. "
```

File: examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb [hunk 4]
```diff
@@ -0,0 +1,434 @@
...
+    "    \n",
+    "print(f\"Opening a connection to {druid_host}.\")\n",
+    "druid = druidapi.jupyter_client(druid_host)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Run the following cell to define the two datasources the tutorial uses, create a SQL client to run SQL, and create a `druidapi` display client to format results."
```
Comment (techdocsmith) [inline examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb:109]: Nit: Do we ever use the variables for the datasources? For example in the INSERT statement, we're just explicitly naming the datasource:
```
INSERT INTO \"kttm\"\n",
```
I'm almost for just removing the variables right now and always just putting in the datasource where we need to. You have to submit the sql a different in druidapi in order to pass the datasource as a parameter.

File: examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb [hunk 5]
```diff
@@ -0,0 +1,434 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Working with nested columns\n",
```
Comment (techdocsmith) [inline examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb:7]: What about "Ingesting and querying data in nested columns"? or something similar?

File: examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb (hunk 4 above)
Comment (writer-jill) [reply examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb:109]: I'll update this in current PR: https://github.com/apache/druid/pull/14788

File: examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb (hunk 3 above)
Comment (writer-jill) [reply examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb:84]: This is an old PR and this doesn't appear in the new one: https://github.com/apache/druid/pull/14788

File: examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb (hunk 1 above)
Comment (writer-jill) [reply examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb:30]: Added to https://github.com/apache/druid/pull/14683

File: examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb (hunk 2 above)
Comment (writer-jill) [reply examples/quickstart/jupyter-notebooks/notebooks/02-ingestion/02-working-with-nested-columns.ipynb:44]: Added to https://github.com/apache/druid/pull/14683

**Pull request closed without merging**